### DISCUSSION TITLE
Can manual labeling improve the accuracy of automatic recognition?

### DISCUSSION THREAD

**Discussion by jerry-yuan** [photoprism/photoprism#3149] [Q&A]
I have some photos managed by photoprism, I found some faces and objects was detected by photoprism with unsatisfactory accuracy. The photoprism has provided a method that I can rectify the recognize results. 

I wonder if my rectify action can improve the face/object recognition accuracy of photos which will be added in future?

**Comment by graciousgrey** [comment]
At the moment, it is not possible to train the image classification (labels).

The face clusters are constantly updated in the background as you make changes, such as adding names or correcting incorrect matches.
